descriptors/ModuleDescriptor-template.json

```diff
@@ -225,7 +225,7 @@
     },
     {
       "id": "location-units",
-      "version": "1.1",
+      "version": "2.0",
```

Comment:
This version change should also be made in the RAML file

Reply:
@marcjohnson-kint  Thank you! I changed this version to 2.0 in the "locationunit.raml"-file.